python/cuml/cuml/tests/test_svm.py

```diff
@@ -464,8 +461,7 @@ def test_svm_memleak(
 
     """
     X_train, X_test, y_train, y_test = make_dataset(dataset, n_rows, n_cols)
-    stream = cuml.cuda.Stream()
-    handle = cuml.Handle(stream=stream)
+    handle = cuml.Handle()
```

Comment:
Do we even need to instantiate the handle ourselves instead of relying on `SVC(handle=None)`?

Reply:
I don't think so, but opted to leave it as is for now. This whole test is currently xfailed for other reasons.

Reply:
Looks like we call `handle.sync` later on after `del svc` - so I guess we need it